x-team/xp-ui
# Update Footer

**Release Type:** *Breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3370

## Description

Update Footer and drop support to previous version.

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3370-update-main-footer` | [XP](https://github.com/x-team/xp/pull/2037)
`XP-3370-update-main-footer` | [XP Registration](https://github.com/x-team/xp-registration/pull/134)

## Steps to Test or Reproduce

> Make sure to test both desktop and mobile viewports

- http://localhost:9001/?path=/story/ui-components-footer--basic-usage
- http://localhost:9001/?path=/story/screens-and-layouts-jobspagelayout--basic-usage
- http://localhost:9001/?path=/story/screens-and-layouts-jobspagelayout-use-cases--job-details-page
- http://localhost:9001/?path=/story/screens-and-layouts-jobspagelayout-use-cases--my-applications-page
- http://localhost:9001/?path=/story/screens-and-layouts-jobspagelayout-debug--job-details-page-applied

## Impacted Areas in Application

- `Footer` got updates
- `FooterBrands` got removed
- `FooterList` got removed
- `JobsPageLayout` now contain the `Footer` and got some updates in the bottom spacing
-  `src/styles/logo.js` now contain 2 variations, one is the X-Team **brand** and the other is the X-Team **logo** (brand and logo are distinct things, not they reflect what they are in fact), the only existing usage was in `HeaderBar` and got updated appropriately
